wasm/wazeroir/interpreter.go

```diff
@@ -112,6 +112,11 @@ type interpreterOp struct {
 	f      *interpreterFunction
 }
 
+// PreCompile Implements wasm.Engine for interpreter.
```

Comment:
is this doc valid? (does nothing)

Reply:
https://github.com/tetratelabs/wazero/pull/60/commits/5fd13b36250f0fb937bfc3867b52f4848d7f0022